common/appscale/common/service_stats/stats_manager.py

```diff
@@ -297,9 +301,13 @@ def _increment_counters(self, counters_config, counters_dict, request_info):
           continue
 
       if isinstance(counter_pair[0], str):
-        # Increment single counter if key is str
         counter_name = counter_pair[0]
-        counters_dict[counter_name] += 1
+        if isinstance(counter_pair[1], summarizers.AttrValue):
```

Comment:
`summarizers.AttrValue` should not be only supported summarizer. `summarizers.Summarizer` should be used here instead.